### PR TITLE
Edit DG

### PR DESCRIPTION
**Pull request by lamyuewei**
As per header

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -2,12 +2,27 @@
...
+        4. [Design Considerations](#12-design-considerations)
+        5. [Aspect: How RecordMealCommand executes and save results](#aspect-how-recordmealcommand-executes-and-save-results)
+        6. [Aspect: Data structure to support the command](#aspect-data-structure-to-support-the-command)
+3. [Product Scope](#product-scope)
+    1. [Target user profile](#target-user-profile)
+    2. [Value proposition](#value-proposition)
+4. [User Stories](#user-stories)
+5. [Non-Functional Requirements](#non-functional-requirements)
+6. [Glossary](#glossary)
+7. [Instructions for Manual Testing](#instructions-for-manual-testing)
```
Comment (yuxianglim) [inline docs/DeveloperGuide.md:25]: OK, thks for updating the content table

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -105,17 +120,17 @@ The `Model` component
...
-The Food consists of the following classes: 
-* `LogsCentre` - Tracks system through log records and saves them to a log file
-* `MessageBank` - Consists of multiple system output messages
-* `Weekday` - Enumeration class for classifying weekdays
-* `"ABC"Exception` - A collection of exceptions to aid in running of the application.
+The Commons consists of the following classes: 
+* `LogsCentre` - Tracks system through log records and saves them into a log file
+* `MessageBank` - Consists of multiple standard system output messages for UI to print
+* `Weekday` - Enumeration class for classifying all 7 possible days in a week
+* `"ABC"Exception` - A collection of exceptions to aid in running of the application
```
Comment (yuxianglim) [inline docs/DeveloperGuide.md:127]: Okay. Good to be more precise

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -190,6 +206,7 @@ Diet Manager aims to achieve the following:
 |v1.1| student|add food items to the database|do not have to constantly check food nutritional value for common foods|
 |v1.2| student|save my diet history|have a record of my daily food intake|
 |v1.2| student|import my diet history|have access to previous records and be able to progress from there|
+|v1.2| student|export my diet history|can view my previous records on other devices and never ever lose my progress|
 |v1.2| student|receive workout advice based on my excess calorie intake for the day|can maintain my calories for the day|
 |v2.0| student|generate a recommended food plan|know what to eat to meet recommended caloric intake|
```
Comment (yuxianglim) [inline docs/DeveloperGuide.md:212]: Good to go. thks for proofreading the DG

**Review by yuxianglim — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (lamyuewei) [reply docs/DeveloperGuide.md:25]: no problem!!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (lamyuewei) [reply docs/DeveloperGuide.md:127]: suree

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (lamyuewei) [reply docs/DeveloperGuide.md:212]: ahahah, no problem! thanks for approval!

**Pull request merged**